### PR TITLE
Added mime analysis pages

### PR DESCRIPTION
**Pull request by antrromet**
Added the MIME analysis of the POLR dump files.

### CODE REVIEW

**Comment by chrismattmann** [comment]
thank you @antrromet appreciate it!

@lorsposto @luuj team, can you bring this up to date and merge?

**Comment by lorsposto** [comment]
hey @antrromet, the changes here are hard to follow, and among them I can't seem to find where you've included your visualizations. I found the files but I don't see them linked anywhere, or they're just hard to find amongst all of the other code changes.

Could you resubmit this with only your added data files, your added visualization html pages, and then add a link to them from /html/d3-examples.html? This will be easier to follow, and from there we can take on reorganization.

Additionally, your refactoring changes are welcome and appreciated (the jQuery header/footer handling is a good idea). If you still want to contribute, please submit them as separate PRs from the latest version of gh-pages!
